app/models/user.rb

```diff
@@ -8,4 +9,11 @@ class User < ApplicationRecord
   validates :password, presence: true, length: {minimum: Settings.validates.user.minimum_pass}
 
   has_secure_password
+  class << self
+    def digest string
+      cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST :
+        BCrypt::Engine.cost
```

Comment:
sao lại xuống dòng trong toán tử ba ngôi nhỉ?

Reply:
:(